SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -10,7 +10,7 @@ Plot {
 	var valueCache;
 
 	*initClass {
-		if(Platform.hasQt.not) { ^nil; };	// skip init on Qt-less builds
+		// if(Platform.hasQt.not) { ^nil; };	// skip init on Qt-less builds
```

Comment:
if it's not needed, perhaps better remove and explain why it can be removed?

Reply:
@telephon i was just experimenting with whether this avoided the break, I wasn't planning to keep this change. in any case it didn't appear to help, so i'm unsure what exactly is going wrong

Reply:
ah good!